src/app/branded/step-1/branded-checkout-step-1.tpl.html

```diff
@@ -44,6 +44,40 @@ <h3 class="panel-name" translate>{{'PAYMENT'}}</h3>
       </checkout-step-2>
     </div>
   </div>
+  <div class="panel" ng-if="$ctrl.premiumCode">
+    <div class="panel-body">
+      <h3 class="panel-name" translate>{{'CHOOSE_RESOURCE'}}</h3>
+      <div class="row text-left">
```

Comment:
As I tested other radio button possibilities in that scenario, I think what might actually be best is to add a style like this:

```
branded-checkout .radio-inline {
  text-align: left;
}
```

My earlier suggestion does not help the payment subtype radio buttons, nor would it help the custom giving amounts.

Reply:
The CSS change fixes the radio button being in the middle of the text. But the div will still be centered and cause the radio buttons to be misaligned if a premium has a short name.

So I will just add the `text-left` class.

Reply:
Sounds good.